tests/unit/amo/server/test_base.js

```diff
@@ -507,6 +507,23 @@ describe(__filename, () => {
       );
     });
 
+    it('catches a URIError from _createHistory and returns a 404', async () => {
```

Comment:
Add a link to the issue please either above this test case or in the `base.j` file because we need to remember why we have this special error handling.